packages/nvidia_nat_app/tests/executors/test_runner.py

```
@@ -0,0 +1,426 @@
+        is_cycle_exit=False,
+    )
+
+
+async def _slow_node(name: str, delay: float = 0.05, result: str | None = None) -> dict:
+    await asyncio.sleep(delay)
+    return {f"{name}_out": result or f"{name}_done"}
+
+
+class _RaisingResolution:
+
+    def resolve(self, label: str):
```

Ruff (0.15.2)

[warning] Unused method argument: `label`

(ARG002)